examples/23_plotly.py

```diff
@@ -0,0 +1,108 @@
+"""Plotly.
+
+Examples of visualizing plotly plots in Viser."""
+
+import time
+
+import numpy as onp
+import plotly.express as px
+import plotly.graph_objects as go
+import viser
+from PIL import Image
+
+server = viser.ViserServer()
+
+# We want to showcase multiple types of plots, so we'll have a dropdown to switch between them.
+# Note that this isn't an exhaustive list of plot types!
+# We should be able to support all plots in https://plotly.com/python/.
+plot_dropdown = server.add_gui_dropdown(
+    "Plot",
+    options=["Line", "Image", "3D Scatter"],
```

Comment:
as an effect of mutability vs immutability the types can be stronger here if we switch this from a list:
![image](https://github.com/nerfstudio-project/viser/assets/6992947/3ddf7137-f1b1-46e6-8d9c-0185ce852da4)

to a tuple:
![image](https://github.com/nerfstudio-project/viser/assets/6992947/d7974f1e-ae17-4ca2-89ae-30d235c571dc)

which is nice because `plot_drop.value = "some value that's not in the options list"` will give a type error

Reply:
Ok! should be addressed in https://github.com/nerfstudio-project/viser/blob/3af36b1bb606bbb26bd59737c4d44eb6333cecc7/examples/23_plotly.py#L20.